test/0133/unknown-optional-params.test.js

```diff
@@ -85,6 +85,13 @@ test('aep-133-unknown-optional-params should find no errors', () => {
                 type: 'string',
               },
             },
+            {
+              name: 'force',
```

Comment:
should this be in 133?

Reply:
Yes. This is an improvement that I discovered when I was writing the test for the 134 variant of this rule.